src/pe_reports/data/db_query.py

```diff
@@ -13,7 +13,12 @@
 
 from .config import config
 
-logging.basicConfig(format="%(asctime)-15s %(levelname)s %(message)s", level="INFO")
+# Setup logging to central file
+logging.basicConfig(
+    filename="pe_reports_logging.log",
```

Comment:
How come you are not using the `CENTRAL_LOGGING_FILE` variable here? Also why are you using `filemode="a"` and different formatting options in `src/pe_reports/__init__.py`, but not here?

Reply:
@dav3r 
I have updated the logging filename and formatting to match throughout the project per comit [7ccf5c9](https://github.com/cisagov/pe-reports/pull/169/commits/7ccf5c9c137ff89f5102120ec00d4ba80ff2c71d) I use the filemode='a' to append the file and as I understand this is what the mode that we want so that we have a continually growing log file. Do you have a recommendation as to another filemode that we should be using?

Reply:
👍  (see https://github.com/cisagov/pe-reports/pull/169#discussion_r952796014 for response).